camus-etl-kafka/src/test/java/com/linkedin/camus/etl/kafka/mapred/EtlInputFormatTest.java

```diff
@@ -0,0 +1,26 @@
+package com.linkedin.camus.etl.kafka.mapred;
+
+import java.util.Arrays;
+
+import org.apache.hadoop.conf.Configuration;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class EtlInputFormatTest {
```

Comment:
We actually just created this class 'EtlInputFormatTest', you can either combine your code with that class or rename your class.